### ISSUE TITLE
"File not found" for JAVA file with Package - Script

### ISSUE THREAD

**Issue by cmdshorty** [atom-community/atom-script#1283]
I'm not sure why this is happening as I have the latest JDK installed and added the PATH in my Environment Variables. But this is the error I'm getting in Atom AND in my Command Prompt:

    javac: file not found: Lab1Main.java
    Usage: javac <options> <source files>
    use -help for a list of possible options
    [Finished in 0.386s]

JAVA is all extremely new to me (only 2 weeks into the college course I'm taking currently) so I'm at the very bottom of knowing what to do here. (Not to mention, I've barely used Atom.) Do I need to set a PATH variable within Atom as well? I would really like to use Atom over Netbeans as it seems very easy to get things done.

Here's a screen shot of my PATH variables.
![capture](https://cloud.githubusercontent.com/assets/24630300/24083023/3e5d06ee-0c9d-11e7-8358-1b379f274585.JPG)

**Comment by JerryGoyal** [comment]
Even I got the same error. Seem like it's not getting the correct path of the java file. I'm opening from the atom icon.. could that be the cause?
```
[Command: cmd '/c javac -Xlint PalindromeExample.java && java PalindromeExample']
javac: file not found: PalindromeExample.java
Usage: javac <options> <source files>
use -help for a list of possible options
[Finished in 0.29s]
```

**Comment by stevenkuipers** [comment]
Could you check if there are any spaces in the directory names and/or filenames? You can check this by running the script and then showing the output in the new tab (see image below for icon).

<img width="1027" alt="screen shot 2018-04-08 at 12 35 01" src="https://user-images.githubusercontent.com/13149753/38466501-657f1930-3b2a-11e8-8c43-928d3b01e913.png">

On mac (where I'm working on, having files and directories with spaces in them is no problem) but earlier this week I assisted someone with the same error: "file not found" on windows. Renaming the folder that had a space in it was a quick fix and afterward, it worked just fine. Only problem is that I don't have windows machine available so I can't do any further testing.

**Comment by dariens** [comment]
I was able to get rid of this error by changing the 'Default Current Working Director (CWD) Behavior' to 'Directory of the script', in the 'script' Settings.

**Comment by davidballad** [comment]
> 
> 
> Could you check if there are any spaces in the directory names and/or filenames? You can check this by running the script and then showing the output in the new tab (see image below for icon).
> 
> <img alt="screen shot 2018-04-08 at 12 35 01" width="1027" src="https://user-images.githubusercontent.com/13149753/38466501-657f1930-3b2a-11e8-8c43-928d3b01e913.png">
> 
> On mac (where I'm working on, having files and directories with spaces in them is no problem) but earlier this week I assisted someone with the same error: "file not found" on windows. Renaming the folder that had a space in it was a quick fix and afterward, it worked just fine. Only problem is that I don't have windows machine available so I can't do any further testing.

do you know how to get rid of the spaces?

**Comment by aminya** [comment]
Should be fixed in #2470. Let me know if it is not.
